eth2util/eth2exp/attagg.go

```diff
@@ -149,3 +159,79 @@ type BeaconCommitteeSubscriptionResponse struct {
 	// IsAggregator indicates whether the validator is an attestation aggregator.
 	IsAggregator bool
 }
+
+// CalculateCommitteeSubscriptionResponse returns a BeaconCommitteeSubscriptionResponse with IsAggregator field set to true if the validator is an aggregator.
+func CalculateCommitteeSubscriptionResponse(ctx context.Context, eth2Svc eth2client.Service, subscription BeaconCommitteeSubscription) (BeaconCommitteeSubscriptionResponse, error) {
+	eth2Cl, ok := eth2Svc.(eth2Provider)
+	if !ok {
+		return BeaconCommitteeSubscriptionResponse{}, errors.New("invalid eth2 service")
+	}
+
+	committeeLen, err := getCommitteeLength(ctx, eth2Cl, subscription.CommitteeIndex, subscription.Slot)
+	if err != nil {
+		return BeaconCommitteeSubscriptionResponse{}, err
+	}
+
+	isAgg, err := IsAggregator(ctx, eth2Cl, int64(committeeLen), subscription.SlotSignature)
+	if err != nil {
+		return BeaconCommitteeSubscriptionResponse{}, err
+	}
+
+	return BeaconCommitteeSubscriptionResponse{
+		ValidatorIndex: subscription.ValidatorIndex,
+		IsAggregator:   isAgg,
+	}, nil
+}
+
+// IsAggregator returns true if the signature is from the input validator. https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/validator.md#aggregation-selection
```

Comment:
I do not understand the comment: "IsAggregator returns true if the signature is from the input validator"